third_party/workspace.bzl

```diff
@@ -138,3 +138,12 @@ def tensorboard_workspace(name = ""):
         ],
         licenses = ["notice"],  # Apache 2.0
     )
+
+    java_import_external(
+        name = "org_jsoup_external",
+        jar_sha256 = "436adf71fe9f326e04fe134cd2785b261f0f4b9b60876adda1de3b6919463394",
+        jar_urls = [
+            "https://jsoup.org/packages/jsoup-1.21.1.jar",
```

Comment:
Consider adding https://repo1.maven.org/maven2/org/jsoup/jsoup/1.21.1/jsoup-1.21.1.jar, since we seem to use this repository in other places.

Reply:
I've gone ahead and added maven and the tf mirror as additional sources